src/Events/Category_Colors/Settings/Settings.php

```diff
@@ -0,0 +1,233 @@
+			);
+		}
+
+		return $field_start;
+	}
+
+	/**
+	 * Registers the Category Colors tab to the settings page.
+	 *
+	 * @since  TBD
+	 *
+	 * @param Tribe__Settings_Tab $display_tab The display settings tab.
+	 *
+	 * @return void
+	 */
+	public function register_tab( Tribe__Settings_Tab $display_tab ): void {
+		// Only load the tab for event settings, not ticket settings.
+		if ( ! tribe( Admin_Settings::class )->is_tec_events_settings() ) {
+			return;
+		}
+
+		// Create the tab instance under "Display".
+		$display_category_colors_tab = new Tribe__Settings_Tab(
+			self::$tab_slug,
+			esc_html__( 'Category Colors', 'the-events-calendar' ),
+			[
+				'priority' => 5.30,
+				'fields'   => apply_filters(
+					'tec_events_settings_display_category_colors_section',
+					$this->generate_settings()
+				),
+			]
+		);
+
+		/**
+		 * Fires after the display category colors settings tab has been created.
+		 *
+		 * @since TBD
+		 *
+		 * @param Tribe__Settings_Tab $display_category_colors_tab The display category colors settings tab.
+		 */
+		do_action( 'tec_events_settings_tab_display_category_colors', $display_category_colors_tab );
+
+		$display_tab->add_child( $display_category_colors_tab );
+	}
+
+	/**
+	 * Generates the settings for the Category Colors feature.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The structured settings array for Category Colors.
+	 */
+	public function generate_settings(): array {
+		$category_color_title = new Div( new Element_Classes( [ 'tec-settings-form__header-block', 'tec-settings-form__header-block--horizontal' ] ) );
+		$category_color_title->add_child(
+			new Heading(
+				_x( 'Category Colors', 'Category Colors section header', 'the-events-calendar' ),
+				3,
+				new Element_Classes( 'tec-settings-form__section-header' )
+			)
+		);
+		$category_color_title->add_child(
+			( new Paragraph( new Element_Classes( 'tec-settings-form__section-description' ) ) )->add_children(
+				[
+					new Plain_Text( __( 'Category colors helps your users relate events to a specific category on the event listing pages. To assign colors to event categories see Event Categories page.', 'the-events-calendar' ) ),
+				]
+			)
+		);
+
+		$category_color          = [
+			'tec-events-category-colors' => $category_color_title,
+		];
+		$category_colors_section = [
+			'category-color-header'                 => ( new Div( new Element_Classes( [ 'tec-settings-form__header-block' ] ) ) )->add_children(
+				[
+					new Heading(
+						_x( 'Category Legend', 'Category Legend settings section header', 'the-events-calendar' ),
+						2,
+						new Element_Classes( [ 'tec-settings-form__section-header' ] )
+					),
+					( new Field_Wrapper(
+						new Tribe__Field(
+							'categoryLegendExplanation',
+							[
+								'type' => 'html',
+								'html' => esc_html__( 'The category legend provides labels for the colors that appear on your events on event listing pages.', 'the-events-calendar' ),
+							]
+						)
+					) ),
```

Comment:
Do you need this as a wrapped field? It can be just plain text here.

```suggestion
					( new Plain_Text( esc_html__( 'The category legend provides labels for the colors that appear on your events on event listing pages.', 'the-events-calendar' ) ),
```